roles/splunk_common/tasks/main.yml

```diff
@@ -32,6 +32,12 @@
     - splunk.allow_upgrade is defined
     - splunk.allow_upgrade | bool
 
+- name: "Set splunk-launch.conf options"
```

Comment:
Ah I didn't see this in this PR initially - this feature should be added already in line 77:
```
- include_tasks: set_launch_conf.yml
  when:
    - "'launch' in splunk and splunk.launch"
```

Reply:
Yeah I'm learning to make my PR's smaller. I think I added this while someone else was working on the same thing and since my method was rolled into this big PR it wasn't noticed.

Reply:
No worries, appreciate the contributions :)